test/e2e/emissary_test.go

```diff
@@ -154,3 +155,16 @@ func TestCPackagesEmissaryVSphereKubernetes122BottleRocketWorkloadCluster(t *tes
 	test := setupSimpleMultiCluster(t, provider, v1alpha1.Kube122)
 	runCuratedPackageEmissaryRemoteClusterInstallSimpleFlow(test)
 }
+
+func TestCPackagesEmissaryNutanixKubernetes122SimpleFlow(t *testing.T) {
```

Comment:
we should remove this test from this PR as i have added for all versions of k8s in https://github.com/aws/eks-anywhere/pull/4231

Reply:
I can remove this